docs/introduction/introduction.md

```diff
@@ -1,109 +1,5 @@
-
-<h2 id="tools">Tools</h2>
-
-The Mbed product suite includes the tools you need to work with Mbed OS, whatever your skill level. If you are an experienced developer with a desktop setup, you may prefer working offline with Arm Mbed CLI, our Python-based command-line tool. You can use Mbed CLI with one of three supported toolchains: Arm Compiler 6, GCC and IAR. You can also [export projects](../tools/exporting.html) for other IDEs, such as Keil MDK. Mbed OS includes integration code for each supported toolchain to make it thread safe.
-
-If you prefer to work online, use the Arm Mbed Online Compiler, our online development tool, which lets you write and build applications using a web browser with no additional setup.
-
-You can use our debugging tools, DAPLink and pyOCD, to program and debug many devices. At the end of the development cycle, you can use the Mbed OS validation tools, Greentea and utest, to test your project.
-
-<h2 id="the-docs">The documentation</h2>
-
-This is the technical documentation for Mbed OS. We have three types of documents: references, tutorials and porting guides. Our references are background technical material about our APIs, architecture and runtime execution. Our tutorials are step-by-step instructions that show you how to perform specific tasks and solve problems. Our porting guides show our silicon Partners how to port Mbed OS to their targets.
-
-Please see the following sections of our documentation for more information.
-
-- [APIs](../apis/index.html).
-- [Quick start](../quick-start/index.html), a guide to creating your first application.
-- [Reference](../reference/index.html), architectural information about our code.
-- [Tools](../tools/index.html), reference material about the tools Mbed OS uses.
-- [Tutorials](../tutorials/index.html).
-- [Going to production](../mbed-os-pelion/going-to-production.html), the process of connecting Mbed OS devices to Pelion Device Management for large-scale production.
-- [Contributing](../contributing/index.html), guidelines about contributing to our open-source project.
-- [Porting guides](../porting/index.html), for Partners and developers interested in porting targets to Mbed OS.
-
-<h2 id="docs-updates">Recently updated documentation</h2>
-
-- A guide to [porting a custom board](../porting/porting-a-custom-board.html).
-- A porting guide for the [static pin map extension](../porting/static-pinmap-port.html).
-- The [UserAllocatedEvent](../apis/userallocatedevent.html) API reference.
-- A [power optimization tutorial](../tutorials/power-optimization.html) about measuring and reducing power consumption.
-- A [mesh tutorial](../tutorials/mesh-tutorial.html) about selecting and using a mesh protocol.
+When I merge into the branch, it turns up in the PR.
```

Comment:
This came from a second commit to the branch. 
Working this way allows me to respond to the PR while working in the comfort of an offline editor.